modules/oadp-kubevirt-setting-resource-limits-and-requests.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-kubevirt-setting-resource-limits-and-requests_{context}"]
+= Velero CPU and memory resource allocations for {VirtProductName}
+
+You set the CPU and memory resource allocations for the `Velero` pod by editing the  `DataProtectionApplication` custom resource (CR) manifest.
+
+.Prerequisites
+
+* You must have the OpenShift API for Data Protection (OADP) Operator installed.
+
+.Procedure
+
+* Edit the values in the `spec.configuration.velero.podConfig.ResourceAllocations` block of the `DataProtectionApplication` CR manifest, as in the following example:
++
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: <dpa_sample>
+spec:
+...
+  configuration:
+    velero:
+      podConfig:
+        nodeSelector: <node selector> <1>
+        resourceAllocations: <2>
+          limits:
+            cpu: "1"
+            memory: 1024Mi
+          requests:
+            cpu: 200m
+            memory: 256Mi
```

Comment:
Are those coming from scale teams?

Reply:
So these values are coming from the Virt docs - https://docs.openshift.com/container-platform/4.16//virt/backup_restore/virt-installing-configuring-oadp.html

Reply:
Removed